x-pack/plugins/osquery/public/shared_components/osquery_results/types.ts

```diff
@@ -5,25 +5,20 @@
  * 2.0.
  */
 
-import type { EcsSecurityExtension } from '@kbn/securitysolution-ecs';
+import type { Ecs } from '@kbn/cases-plugin/common';
```

Comment:
Is there a reason why we use now the cases `Ecs` ?

Reply:
The reason behind putting ECS data to Osquery in the first place is actually to send these data back to Cases... 
From what I've been searching for there is not one universal ECS interface, and Cases is the smallest one that meets our requirements, so was most convenient to be used.